README.md

```diff
@@ -31,7 +31,8 @@ Repository for you to raise a Pull Request to **practice** open-source! 🎉
 *_You CAN make edits on the main branch, but this may cause issues down the line. Best practice is to create a new branch for each separate issue you work on. That way your `main` branch remains in sync with Eddie's `main` branch_
 
 3. Edit: 
-- On the top right of the Readme file, click on the pencil icon to edit the file by adding your name
+- On the top right of the Readme file, click on the pencil icon to edit the file by adding your name.
+- After editing the Readme file, add a commit message and click on the green button saying "Commit Changes". Make sure you have selected the correct branch.
```

Comment:
Please create new issue and PR for these changes :)

Reply:
Here maybe some small explanation why smaller and focused PRs are likely to be better.
## Smaller and focused Pull Requests (PR)

### TLDR;

The smaller and more focused the PR the easier for the reviewer and the more likely it is to to merge faster.

---

### Why are smaller PRs better?

- Easier for the reviewer
- Will get merged faster
- Reduces conflicts with other changes
- You get more green squares if you separate PRs out into focused changes


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Inefficient